nvflare/fuel/utils/component_builder.py

```diff
@@ -28,6 +27,20 @@ def get_module_scanner(self):
         """
         pass
 
+    def is_class_config(self, config_dict: dict) -> bool:
+        # the dict has both path/name or args, it's likely a component,
+        # test if it has a valid class path
+        if "args" in config_dict and ("path" in config_dict or "name" in config_dict):
```

Comment:
the comment string should be "the dict has both args and path/name"

What if the SUB component does not specify args?

So maybe somewhere in documentation we need to say you need to specify "args" for it to be treat as a SUB component?